### PR TITLE
Move instructor notes inline and add some more

### LINKED ISSUE #146: Move Instructor notes to appropriate episodes as inline notes

### PR DESCRIPTION
**Pull request by bencomp**
This fixes #146. Some notes from the instructor notes page are now inline notes, though some general notes are still in the separate page. All instructor notes and callouts now have a heading, to allow assessing their value without having to view the contents first.

The Introduction episode (or chapter) now starts with a call for contributions from instructors with links to issues.

Since the index.md and setup.md files are merged into the front page of the lesson, some references had to be removed or updated to make sense in the new context.

### CODE REVIEW

**Comment by bencomp** [comment]
Note that I don't know if https://github.com/carpentries/sandpaper/issues/470 is related to the contents of this PR...

File: learners/setup.md [hunk 1]
```diff
@@ -59,6 +91,8 @@ OpenRefine is safe to run.
    selecting "Extract…". Name that directory something like OpenRefine.
    
    :::::::::::::::::::::::::::::::::::::::::  callout
+   #### Long paths
```
Comment (tobyhodges) [inline learners/setup.md:94]:
```suggestion
   
   ## Long paths
```

I don't think this extra newline is _necessary_ but I do think it makes the source files more readable and it makes this box consistent with the rest of your lesson. We really need to start a style guide for writing lesson source Markdown...

**Review by tobyhodges — APPROVED** [review]

**Comment by tobyhodges** [comment]
I made a couple of comments but neither are important enough to block merging if you want to.

File: learners/setup.md [hunk 2]
```diff
@@ -22,6 +31,18 @@ for this lesson.
...
+
+### About the data / Import from URL
+
+A general description of the dataset used in the Social Sciences lessons can
+be found [in the workshop data home page](https://www.datacarpentry.org/socialsci-workshop/data/).
+
+Instead of downloading the data to the computer, you could import the data from
+the URL directly when you start the project.
+When learners have trouble finding the file on their computer, this may be a
+workaround to not have to wait.
```
[the remark this answers is not in the document]
Comment (bencomp) [reply learners/setup.md:44]: No, I haven't tried that. It does sound like a reasonable thing to do here.

**Comment by bencomp** [comment]
thanks for the review and suggestions, @tobyhodges. I tried to make you co-author on the additional commits, but used the wrong trailer 🤦

**Pull request merged**